link.go

```diff
@@ -41,6 +41,18 @@ type LinkAttrs struct {
 	NetNsID      int
 	NumTxQueues  int
 	NumRxQueues  int
+	Vfs          []VfInfo // virtual functions available on link
```

Comment:
@jellonek Is LinkAttr the only place or is it the proper place where we can pass these virtual functions ? Can you think of another way to expose this setting ?
I am asking because this would break the existing API.

Reply:
True, this is addition, we should be fine. Thanks.